packages/netlify-cms-backend-git-gateway/src/implementation.ts

```diff
@@ -303,32 +293,7 @@ export default class GitGateway implements Implementation {
   }
 
   getMedia(mediaFolder = this.mediaFolder) {
-    return Promise.all([this.backend!.getMedia(mediaFolder), this.getLargeMediaClient()]).then(
```

Comment:
All pointer files were retrieved on media library load. Change it to lazy load the pointer file when `getMediaDisplayURL` is called